src/commands/recipes/recipes.ts

```diff
@@ -26,8 +33,10 @@ export const recipesCommand = async (recipeName: string, options: OptionValues,
     return command.help()
   }
 
+  const args = command.args.slice(1)
+
   try {
-    return await runRecipe({ config, recipeName: sanitizedRecipeName, repositoryRoot })
+    return await runRecipe({ args, command, config, recipeName: sanitizedRecipeName, repositoryRoot })
```

Comment:
We were previously not passing the command arguments to recipes, but it's useful when a recipe needs to take input from the user.